pkg/kubelet/dockershim/network/cni/cni_test.go

```diff
@@ -304,7 +304,7 @@ func TestCNIPlugin(t *testing.T) {
 	}
 	expectedBandwidth := map[string]interface{}{
 		"ingressRate": 1000000.0, "egressRate": 1000000.0,
-		"ingressBurst": 2147483647.0, "egressBurst": 2147483647.0,
+		"ingressBurst": 1000000.0, "egressBurst": 1000000.0,
```

Comment:
Good catch ! but , i  think changing this is the sticking point, which the sig moving the docker provider out of tree will have to own